src/pipelines/vaccinations/config.yaml

```diff
@@ -213,6 +213,16 @@ sources:
       job_group: all
       deferred: true
 
+  # Data sources for IL levels 2 + 3
+  - name: pipelines.epidemiology.il_authority.IsraelDataSource
+    fetch:
+      - url: "https://data.gov.il/api/3/action/datastore_search?resource_id=d07c0771-01a8-43b2-96cc-c6154e7fa9bd"
+    test:
+      location_key_match: 'IL.*'
```

Comment:
Please add more specific test matches, I recently learned that this would match for example `US_IL_*`.
```suggest
      location_key_match:
        - '^IL$'
        - '^IL_[^_]+$'
        - '^IL_[^_]+_[^_]+$
```

Reply:
Ah good call. I might fix this in epidemiology as well then.

Reply:
Done.